kaitai/kaitaistream.cpp

```diff
@@ -635,9 +635,10 @@ uint8_t kaitai::kstream::byte_array_max(const std::string val) {
 #include <iconv.h>
 #include <cerrno>
 #include <stdexcept>
+#include "kaitaistream.h"
 
-std::string kaitai::kstream::bytes_to_str(std::string src, std::string src_enc) {
-    iconv_t cd = iconv_open(KS_STR_DEFAULT_ENCODING, src_enc.c_str());
+std::string kaitai::kstream::bytes_to_str(const std::string src, const char* src_enc) {
```

Comment:
Again, I'd prefer this a bit more (see our previous discussion in https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/pull/57#discussion_r1233123710):

```diff
-std::string kaitai::kstream::bytes_to_str(const std::string src, const char* src_enc) {
+std::string kaitai::kstream::bytes_to_str(const std::string src, const char *src_enc) {
```

...but this is really more like a job for an autoformatter.

Reply:
Fixed, thanks for the reminder. We should definitely set up some clang-format or something like that.